.github/workflows/pushimage-next.yaml

```diff
@@ -14,18 +14,34 @@ on:
 
 jobs:
 
-  build:
+  indexServerBuild:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout registry-support source code
+    - name: Checkout devworkspace-operator source code
       uses: actions/checkout@v2
-    - name: Login to Quay
-      uses: docker/login-action@v1 
+    - name: Docker Build & Push
+      uses: docker/build-push-action@v1.1.0
       with:
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
         registry: quay.io
+        repository: devfile/devfile-index-base
+        dockerfile: ./index/server/Dockerfile
+        tags: next
+        tag_with_sha: true
+
+  ociRegistryBuild:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout devworkspace-operator source code
+      uses: actions/checkout@v2
+    - name: Docker Build & Push
```

Comment:
So that's a good question. The reason I went with this GitHub action is that it allows us to tag each push with a sha, making it easy to pull / refer back to specific images we've pushed to quay, and the push.sh script doesn't allow that.

I can add a push.sh script for local development, but my preference would still be to use this GitHub action, WDYT?

Reply:
Yeah, that's what I mean, for pushing image to quay we still use the GitHub action template/script, just add extra push.sh script for user's manual push.

Reply:
Gotcha 👍

Reply:
Updated.